app/routes/user/account/connected-accounts.$id.tsx

```diff
@@ -0,0 +1 @@
+export { ConnectedAccountDetail as default } from "@clerk/clerk-js/src/ui/userProfile/connectedAccounts/ConnectedAccountDetail";
```

Comment:
This is the main way I could see the library being used. There's unfortunately no way to abstract reusable _routes_ yet, but by creating separate routes for the sub components, devs could add additional loaders, actions, etc. As well as decorating the component with any additional UX

`@clerk/remix` could expose a dev script that makes the initial generation of these files a lot easier, something like `npx clerk remix init`